pkg/activator/config/store.go

```diff
@@ -33,7 +33,11 @@ type Config struct {
 
 // FromContext obtains a Config injected into the passed context
 func FromContext(ctx context.Context) *Config {
-	return ctx.Value(cfgKey{}).(*Config)
+	val, ok := ctx.Value(cfgKey{}).(*Config)
+	if ok {
+		return val
+	}
+	return nil
```

Comment:
Is this change needed? The private cfgKey should make sure we can only set this value via our functions in here and they should make sure it's always the correct type, I'd think.

Reply:
I saw that when our test cases were running, we were getting an error due to tracing config not being set in the context. I *think* tracing config is always in the context during normal execution, so we should be fine without this change -- i updated the test cases to address this.